packages/build/.depcheckrc

```diff
@@ -8,3 +8,6 @@ ignores:
  - eslint-config-mongodb-js
  # transitive type dependency of @types/tmp-promise
  - '@types/tmp'
+ - yaml
```

Comment:
This was, I think, the only thing I had to add to .depcheckrc when I merged in @addaleax 's work. I didn't spend much time trying to work it out, but I don't actually know why it would be new.